parser.js

```diff
@@ -40,7 +41,12 @@ module.exports = {
 
     var output = solc.compileStandard(JSON.stringify(solcStandardInput), function(file_path) {
       // Tell the compiler we have source code for the dependency
-      return {contents: "pragma solidity ^0.4.0;"};
+      path_components = file_path.split(/\/(.+)/);
+      package_name = path_components[0];
+      sol_path = path_components[1];
+
+      dep_path = 'installed_contracts/'+package_name+'/contracts/'+sol_path;
```

Comment:
Looked into the failing test - it looks like this code breaks the situation where you are just importing a local dependency (not one from `installed_contracts`)